components/asset-metadata-service/pkg/fileheader/fileheader.go

```diff
@@ -0,0 +1,41 @@
+package fileheader
```

Comment:
It should be moved to common, as it is also created here: https://github.com/kyma-project/kyma/blob/master/components/asset-upload-service/internal/fileheader/fileheader.go

Reply:
Exactly, that's my intention. But we need to do it in a follow-up task.